contrib/cindercompatibleapi/api/attachment.go

```diff
@@ -39,7 +39,8 @@ type AttachmentPortal struct {
 func (portal *AttachmentPortal) DeleteAttachment() {
 	id := portal.Ctx.Input.Param(":attachmentId")
 	attachment := model.VolumeAttachmentSpec{}
-	err := client.DeleteVolumeAttachment(id, &attachment)
+	NewClient(portal.Ctx)
```

Comment:
Can you explain what's the meaning of this line of code?

Reply:
The tenantId and tokenID are obtained through the information of portal.Ctx. If successful, a new opensdsClient is generated.